File: app/src/main/java/com/example/work_liuchangxu/work_0429/MyLoadingEvent.java
package com.example.work_liuchangxu.work_0429;

public class MyLoadingEvent {

    public static int STATE_LOADING = -1;

    public static int STATE_COMPLETE = -2;
    public static int STATE_END = -3;

    public int LOADING_STATE = 0;

    public MyLoadingEvent(int state){
        this.LOADING_STATE = state;
    }
}

File: app/src/main/java/com/example/work_liuchangxu/work_0430/MyLoadingEvent.kt
package com.example.work_liuchangxu.work_0430

class MyLoadingEvent(state: Int) {
    @JvmField
    var LOADING_STATE = 0
    var current = 0

    init {
        LOADING_STATE = state
    }

    companion object {
        @JvmField
        var STATE_LOADING = -1
        var STATE_COMPLETE = -2
        var STATE_END = -3
    }
}
